web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -1119,22 +1126,10 @@
                     urlGetFeature = gnGlobalSettings.proxyUrl
                                         + encodeURIComponent(urlGetFeature);
                   }
-
-                  $.ajax({
```

Comment:
Let OL handle this instead of doing it manually

Reply:
By discarding the custom loader we lose the `loadingLayer` information. Not sure if that was intended in your PR?